plugins/apitracing/src/lib/FunctionHook.cpp

```diff
@@ -56,24 +57,62 @@ namespace ApiTracing
             }
 
             auto extractedParameters = extractor->extractParameters(event, parameterInformation);
-            logParameterList(extractedParameters);
+
+            Json::Value root;
+            Json::StreamWriterBuilder builder;
+            builder["indentation"] = "";
+            root["functionName"] = functionName;
+            auto json = getParameterListAsJson(extractedParameters, root);
+            std::string unformattedTraces = Json::writeString(builder, json);
+
+            logger->info("Monitored function parameters",
+                         {{"ProcessDtb", fmt::format("{:x}", event.getCr3())},
+                          {"ThreadID", event.getGs()},
+                          {"Traces", unformattedTraces}});
         }
         return BpResponse::Continue;
     }
 
+    Json::Value
+    FunctionHook::getParameterListAsJson(const std::vector<ExtractedParameterInformation>& extractedParameters,
+                                         Json::Value root)
+    {
+        for (const auto& extractedParameter : extractedParameters)
+        {
+            Json::Value parameter;
+            std::visit(
+                [&parameter = parameter, &extractedParameter = extractedParameter](auto&& arg)
+                { parameter[extractedParameter.name] = std::forward<std::remove_reference_t<decltype(arg)>>(arg); },
```

Comment:
To address sonarqube lints you can do:
```suggestion
                [&parameter = parameter, &extractedParameter = extractedParameter]<typename T>(T&& arg)
                { parameter[extractedParameter.name] = std::forward<T>(arg); },
```